jenkins-integration/beaker/install/shared/23_install_system_gems.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+test_name 'Install System Gems'
+
+def install_system_gem(host, gem_name)
+  step "Install '#{gem_name}' gem"
+  # Ensure it's the system ruby
+  on(host, "/usr/bin/env gem install #{gem_name}")
+end
+
+step 'Install gems using system ruby'
+
+gem_list = ENV['PUPPET_GATLING_SYSTEM_GEMS'].split(',')
```

Comment:
Cool, yeah, I'm fine with deferring on exposing this up to the `Jenkinsfile` layer.  I'd be fine with saying that a version is required (not optional) for each gem listed if that works for y'all.